less/index.less

```
@@ -53,7 +53,188 @@ table {
     box-sizing: border-box;
 }
 
+font {
+	font-family: 'Special Elite', cursive;
+	font-size: 62.5%;
+}
+
 html, body {
-    height: 100%;
+	height: 100%;
+}
+
+h1 {
+	font-size: 3.8rem;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+// ======================================= variables
+@text-color: #4C4F56;
+@button-color: #8C7561;
+@divider-color: #ECEDEE;
+@check-border: 0.1px solid red;
+@bkgrd-color: #D8C5A5;
+
+// ======================================= mixins
+.center-all() {
+	width: 100vw;
+	margin: 0 auto;
+}
+
+.center-flex() {
+	display: flex;
+	justify-content: center;
+}
+
+.left-flex() {
+	display: flex;
+	justify-content: flex-start;
+}
+
+.right-flex() {
+	display: flex;
+	justify-content: flex-end;
+}
+
```

```suggestion
.flex(@justify){
display: flex;
justify-content: @justify;
}
```
then you could use `flex(flex-end)` `flex(flex-start)` etc and save you a few dozen lines of code.